packages/web/src/services/audio-player/AudioPlayer.ts

```diff
@@ -426,6 +445,15 @@ export class AudioPlayer {
       (Math.pow(VOLUME_CHANGE_BASE, value) - 1) / (VOLUME_CHANGE_BASE - 1)
   }
 
+  setPlaybackRate = (value: PlaybackRate) => {
+    this.playbackRate = value
+    this._updateAudioPlaybackRate()
+  }
+
+  _updateAudioPlaybackRate = () => {
+    this.audio.playbackRate = parseFloat(this.playbackRate)
```

Comment:
yeah since we have to parse it anyway, it might be good to use number for playback rate everywhere, and then only on the display component we map from number to label?